src/components/NcSelect/NcSelect.vue

```diff
@@ -1008,13 +1008,14 @@ body {
 		}
 	}
 
-	// Hide search from dom if unused to prevent unneeded flex wrap
-	.vs__search[readonly] {
-		position: absolute;
-	}
-	// If search if hidden, ensure that the height of the search is the same
 	.vs__selected-options {
+		// If search if hidden, ensure that the height of the search is the same
```

Comment:
```suggestion
		// If search is hidden, ensure that the height of the search is the same
```
Just a little typo here that could be fixed :)

Reply:
Fixed :)